tests/data_registry/test_views.py

```diff
@@ -28,3 +35,27 @@ def test_detail(self, get_files):
 
         self.assertTemplateUsed("detail.html")
         self.assertContains(response, f"""<a href="{url}" rel="nofollow" download>2022</a>""", html=True)
+
+    def test_publications_api(self):
+        response = Client().get("/en/publications.json")
+        expected_response = {
+            "id": self.collection.id,
+            "title": self.collection.title_en,
+            "source_id": self.collection.source_id,
+            "update_frequency": Collection.UpdateFrequency.UNKNOWN.name,
+            "country": "",
+            "data_to": None,
+            "data_from": None,
+            "frozen": False,
+            "language": "",
+            "last_retrieved": None,
+            "retrieval_frequency": "",
+            "source_url": "",
+            "region": "",
+        }
+        self.assertEqual(response.status_code, 200)
+        self.assertJSONEqual(str(response.content, encoding="utf8"), [expected_response])
```

Comment:
https://docs.djangoproject.com/en/5.0/topics/testing/tools/#django.test.Response.json

Repeat below.

```suggestion
        self.assertJSONEqual(response.json(), [expected_response])
```